lib/src/components/modal/ModalFooterConfirm.js

```diff
@@ -6,7 +6,8 @@ export function ModalFooterConfirm({
   tertiaryText,
   tertiaryOnClick,
   secondaryText,
-  secondaryOnClick
+  secondaryOnClick,
```

Comment:
This may be a separate issue to this PR, but I'm not sure about the naming of the props based on the button module name rather than the button responsibility.

It feels like `confirmText` and `confirmOnClick` would be clearer to understand from the app. The current implementation requires knowledge of what we use our buttons got to make a guess at what props to pass.